src/transforms/transform-pull-request.ts

```diff
@@ -87,40 +85,57 @@ export const transformPullRequest = async (
 	log?: Logger
 ) =>
 {
-	const { head } = pullRequest;
+	const {
+		id,
+		user,
+		comments,
+		base,
+		number,
+		updated_at,
+		head,
+		state,
+		merged_at,
+		title,
+		html_url
+	} = pullRequest;
 
 	const issueKeys = extractIssueKeysFromPr(pullRequest);
 
 	// This is the same thing we do in sync, concatenating these values
 	if (isEmpty(issueKeys) || !head?.repo) {
 		log?.info({
-			pullRequestNumber: pullRequest.number,
-			pullRequestId: pullRequest.id
+			pullRequestNumber: number,
+			pullRequestId: id
 		}, "Ignoring pullrequest since it has no issue keys or repo");
 		return undefined;
 	}
 
+	const branches = await getBranches(gitHubInstallationClient, pullRequest, issueKeys);
+	const author = getJiraAuthor(user, await getGithubUser(gitHubInstallationClient, user?.login));
+	const reviewers = await mapReviews(reviews, gitHubInstallationClient);
+	const status = mapStatus(state, merged_at);
+
 	return {
-		...transformRepositoryDevInfoBulk(pullRequest.base.repo, gitHubInstallationClient.baseUrl),
-		branches: await getBranches(gitHubInstallationClient, pullRequest, issueKeys),
+		...transformRepositoryDevInfoBulk(base.repo, gitHubInstallationClient.baseUrl),
+		branches,
 		pullRequests: [
 			{
 				// Need to get full name from a REST call as `pullRequest.user.login` doesn't have it
-				author: getJiraAuthor(pullRequest.user, await getGithubUser(gitHubInstallationClient, pullRequest.user?.login)),
-				commentCount: pullRequest.comments || 0,
-				destinationBranch: pullRequest.base.ref || "",
-				destinationBranchUrl: `${pullRequest.base.repo.html_url}/tree/${pullRequest.base.ref}`,
-				displayId: `#${pullRequest.number}`,
-				id: pullRequest.number,
+				author,
+				commentCount: comments || 0,
+				destinationBranch: base.ref || "",
+				destinationBranchUrl: `${base.repo.html_url}/tree/${base.ref}`,
+				displayId: `#${number}`,
+				id: number,
 				issueKeys,
-				lastUpdate: pullRequest.updated_at,
-				reviewers: await mapReviews(reviews, gitHubInstallationClient),
-				sourceBranch: pullRequest.head.ref || "",
-				sourceBranchUrl: `${pullRequest.head.repo.html_url}/tree/${pullRequest.head.ref}`,
-				status: mapStatus(pullRequest.state, pullRequest.merged_at),
-				timestamp: pullRequest.updated_at,
-				title: pullRequest.title,
-				url: pullRequest.html_url,
+				lastUpdate: updated_at,
+				reviewers: reviewers.reverse(),
```

Comment:
By reversing the data here instead of in the mapping function, our reviewers still render as they do today (most recent first) whilst also sending the correct approval state.